README.md

```diff
@@ -47,7 +47,28 @@ no further configuration is required.
 
 Otherwise create a dataset and an API token in [Axiom settings](https://app.axiom.co/settings/profile), then export them as environment variables `NEXT_PUBLIC_AXIOM_DATASET` and `NEXT_PUBLIC_AXIOM_TOKEN`.
 
-## Usage
+
+## Capture traffic requests
+
+Create a `middleware.ts` in the root dir of your app:
+
+```typescript
+import { Logger } from 'next-axiom'
+import { NextResponse } from 'next/server'
+import type { NextFetchEvent, NextRequest } from 'next/server'
+
+export async function middleware(request: NextRequest, event: NextFetchEvent) {
+    const logger = new Logger({ source: 'middleware' }); // traffic, request
```

Comment:
what does the comment mean?

Reply:
it was the options we could use instead of `source: "middleware"`. I forgot to remove them. now removed.